contracts/scripts/demo.js

```diff
@@ -53,6 +53,9 @@ async function script() {
   console.log("Token Burned. ID: " + event.args.tokenId.toString());
 }
 
-module.exports = (callback) => {
-  script().then(callback);
-};
+(async function () {
+  await script()
+})();
+// module.exports = (callback) => {
+//   script().then(callback);
+// };
```

Comment:
Please restore the original version. Or at least remove the commented out old version.